app/views/static_pages/community_rules.html.erb

```diff
@@ -35,7 +35,7 @@
         <strong>Asking about or encouraging the use of Windows</strong>
       </li>
       <li>
-        <strong>Discussing mental health issues in depth or seeking mental health help from other users</strong>
+        <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical concerns, homeopathic remedies, drugs, etc.
```

Comment:
We're moving drug related conversation to 5 zap, so I believe mention of them should be removed from here.

Reply:
```suggestion
        <strong>Discussing unrelated topics that can be divisive or harmful</strong> such as politics, religion, relationships, mental health, medical concerns, homeopathic remedies, etc.
```

Reply:
Oh wow, I just re-read the reasoning for this 🤦. I think I'd still remove it as to avoid possible confusion when someone receives a 5 zap and believes it should only be a 0 zap. We can always not zap and approach things gently as you've mentioned.

Reply:
Will remove reference to drugs here.